packages/shared/components/profitability/profitability.tsx

```diff
@@ -9,42 +9,40 @@ import {
   composeProfitabilityPrefix
 } from "./profitability.helper";
 
-interface IProfitabilityProps {
+const renderPrefix = (value: number | string, prefix: PROFITABILITY_PREFIX) => {
+  if (value > 0) return composeProfitabilityPrefix(prefix).positive;
+  if (value < 0) return composeProfitabilityPrefix(prefix).negative;
+};
+
+const Profitability: React.FC<Props> = props => {
+  const {
+    className,
+    value,
+    variant = PROFITABILITY_VARIANT.TEXT,
+    prefix = PROFITABILITY_PREFIX.NO_PREFIX,
+    children
+  } = props;
+
+  return (
+    <BaseProfitability
+      className={className}
+      variant={variant}
+      isPositive={value > 0}
+      isNegative={value < 0}
+    >
+      <>
+        {renderPrefix(value, prefix)}
+        {children}
+      </>
+    </BaseProfitability>
+  );
+};
+
+export default React.memo(Profitability);
```

Comment:
Change to `const Profitability = React.memo(_Profitability); export default Profitability `